src/ExtensionsSample/ExtensionsSample.csproj

```diff
@@ -5,17 +5,21 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <Version>2.0.0-beta1$(VersionSuffix)</Version>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
I realize this was added by the tool, but we should move the LangVersion up into the previous group and out of the conditional

Reply:
Fixed